Java/src/main/java/Planes/Plane.java

```diff
@@ -3,58 +3,58 @@
 import java.util.Objects;
 
 abstract public class Plane {
-    String model;
+
     private int maxSpeed;
     private int maxFlightDistance;
     private int maxLoadCapacity;
 
-    public Plane(String model, int maxSpeed, int maxFlightDistance, int maxLoadCapacity) {
-        this.model = model;
+    String planeModel;
+
+    public Plane(String planeModel, int maxSpeed, int maxFlightDistance, int maxLoadCapacity) {
+        this.planeModel = planeModel;
         this.maxSpeed = maxSpeed;
         this.maxFlightDistance = maxFlightDistance;
         this.maxLoadCapacity = maxLoadCapacity;
     }
 
-    public String getModel() {
-        return model;
+    public String getPlaneModel() {
+        return planeModel;
     }
 
-    public int getMS() {
+    public int getMaxSpeed() {
         return maxSpeed;
     }
 
-    public int Get_Max_Flight_Distance() {
+    public int GetMaxFlightDistance() {
```

Comment:
Method names are lowercase in Java
https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names